capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/video_embed/video-embed.tpl.php

```diff
@@ -0,0 +1,31 @@
+<script type="text/javascript">
```

Comment:
@zero2one I need this piece of js in the tpl, because i need to pass the id of the youtube video to it. as you can [see here](https://github.com/capacity4dev/capacity4more/pull/488/files#diff-74bfb8652085e4ea7ad3aa06365a8575R6), Can i pass it to a js file ?